articles/client-platforms/_includes/_authorization-create-rule.md

```diff
@@ -0,0 +1,9 @@
+First, we will create a rule that assigns our users either an `admin` role, or a single `user` role. To do so, go to the [new rule page](https://manage.auth0.com/#/rules/new) and select the "*Set Roles To A User*" template, under *Access Control*. Then, replace this line from the default script:
```

Comment:
We do not hard-code values like the dashboard URL. Instead we use variables where applicable, in this case you have to use the variable `uiURL` that holds the value `https://manage.auth0.com`.

So instead of `[new rule page](https://manage.auth0.com/#/rules/new)`
you should use `[new rule page](${uiURL}/#/rules/new)`

Please make the update.

For more information: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#document-variables